### PR TITLE
Add version number to chainer module

### PR DESCRIPTION
**Pull request by delta2323**
This PR is implementation of #29. How to get version number is as follows.

```
>>> import chainer
>>> chainer.__version__
'1.0.1'
```

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,7 +16,8 @@
               'chainer.utils'],
     package_data={'chainer.requirements': ['cuda-requirements.txt']},
     install_requires=['numpy',
-                      'six>=1.9.0'],
+                      'six>=1.9.0',
+                      'setuptools'],
```
Comment (beam2d) [inline setup.py]: setup.py imports setuptools, so this dependency is meaningless.

**Comment by beam2d** [comment]
I think the document version should use this version number, too.

**Comment by delta2323** [comment]
@beam2d Thank you for comments! I applied your comments.

**Comment by beam2d** [comment]
LGTM!

**Comment by beam2d** [comment]
Thank you!

**Pull request merged**